tests/v2/validation/Dockerfile.validation

```diff
@@ -19,6 +19,8 @@ RUN go mod download && \
 
 RUN CGO_ENABLED=0
 
+RUN wget https://releases.hashicorp.com/terraform/0.13.7/terraform_0.13.7_linux_amd64.zip && apt-get update && apt-get install unzip &&  unzip terraform_0.13.7_linux_amd64.zip && rm terraform_0.13.7_linux_amd64.zip && chmod u+x terraform && mv terraform /usr/bin/terraform
```

Comment:
[reliability enhancement]:
is this version meant to be kept hard coded? Would we want to make the version a variable instead that a user can pass in, if we end up keeping this as a dockerfile install?

Reply:
This ideally will not be hardcoded.  I intended to circle back to this and add a variable, but was unsure how to supply that value to the freeform job, so for testing, i did not use a variable yet, and yeah.. moving forward, i will figure out how to connect these dots and update to use a variable 👍